### DISCUSSION TITLE
create multiple root location

### DISCUSSION THREAD

**Discussion by afahmy11** [PowerShell/Win32-OpenSSH#2041] [Q&A]
hi everyone

is it possible to use multiple root directories in the config file? the reason is, i am adding several folders to be accessible through SFTP and i don't want to put the root directory as C:\

**Comment by tgauth** [comment]
Hi @afahmy11 - ChrootDirectory can only be specified once in sshd_config. 
Could all the folders that need to be accessible through SFTP be put into a single directory, that is then specified with ChrootDirectory?
